### PR TITLE
Themes 926

### PR DESCRIPTION
**Pull request by blakeganderson**
## Description

This PR adds the text-output-block to the feed-components repo

## Jira Ticket

- [THEMES-926](https://arcpublishing.atlassian.net/browse/THEMES-926)

## Acceptance Criteria

- text output is available in feed-components repo

## Test Steps

1. Checkout this branch `git checkout THEMES-926`
2. In the demo-outboundfeeds repo, run `npx fusion start -f -l @wpmedia/textfile-block,@wpmedia/text-output-block`
3. Verify that the text output block works correctly

[THEMES-926]: https://arcpublishing.atlassian.net/browse/THEMES-926?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: blocks/text-output-block/output-types/text.test.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/**
+ * this is for mocking node env
+ * will not have window attribute, testing ssr
+ * https://jestjs.io/docs/en/configuration.html#testenvironment-string
+ * @jest-environment node
+ */
+import React from "react";
```
Comment (malavikakoppula) [inline blocks/text-output-block/output-types/text.test.jsx]: This file shouldn't use react and enzyme. Can you please update this?

**Review by malavikakoppula — DISMISSED** [review]

File: blocks/text-output-block/output-types/.npmignore [hunk 2]
```diff
@@ -0,0 +1 @@
+*.test.jsx
```
Comment (nschubach) [inline blocks/text-output-block/output-types/.npmignore]:
```suggestion
*.test.jsx

```

Comment (nschubach) [reply blocks/text-output-block/output-types/.npmignore]: minor nitpick, blank EOL recommended

Comment (malavikakoppula) [reply blocks/text-output-block/output-types/.npmignore]: @nschubach Fixed it in [eb27f54](https://github.com/WPMedia/feed-components/pull/654/commits/eb27f54e7678040ec81acdb4308002c9c719b2b0)

**Review by nschubach — APPROVED** [review]
Looks good

**Pull request merged**